src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -139,6 +139,51 @@ Copyright (c) .NET Foundation. All rights reserved.
           TaskParameter="TargetOutputs"
           ItemName="_ProjectReferences" />
     </MSBuild>
+
+    <MSBuild
+      Condition="'$(PackBuildOutput)' == 'true'"
+      Projects="$(MSBuildProjectFullPath)"
+      Targets="BuiltProjectOutputGroup"
```

Comment:
Are XML doc comment files supported?  Are they in this output group or another group?